### ISSUE TITLE
Question: Systctl parameters set in both /etc/sysctl.conf and /etc/sysctl.d/paramname.conf 

### ISSUE THREAD

**Issue by CalvinHartwell** [thias/puppet-sysctl#47]
Hey thias, all, 

I've just started using the module and I've noticed parameter values I set are placed both in /etc/sysctl.conf and /etc/sysctl.d/paramname.conf. 

As /etc/sysctl.conf always has precedence over the conf files in /etc/sysctl.d/, why are they placed in both places? Does this have a benefit or is it for compatibility with multiple operating systems apart from CentOS and RHEL? 

Cheers,

- Calvin

**Comment by shakerky** [comment]
Agree. When $purge is set it should be removing all non-puppet managed values, including those in the above file.
